komga-webui/src/components/bars/ToolbarSticky.vue

```diff
@@ -16,22 +16,25 @@ export default Vue.extend({
   name: 'ToolbarSticky',
   computed: {
     barStyle (): any {
-      if (['xs', 'sm'].includes(this.$vuetify.breakpoint.name)) {
+      if (['xs', 'sm'].includes(this.$vuetify.breakpoint.name as any)) {
         return { 'top': '56px' }
       } else {
         return { 'top': '64px' }
       }
     },
+    barColor (): any {
+      if (this.$vuetify.theme.dark) {
+        return 'grey darken-4'
+      } else {
+        return 'grey lighten-4'
+      }
+    },
   },
   props: {
     elevation: {
       type: Number,
       default: undefined,
     },
-    color: {
```

Comment:
Removing this prop will make the component less reusable. Vuetify offers [custom colors](https://vuetifyjs.com/en/customization/theme/#custom-theme-variants), we could define a base color for each theme, based on white/black, which would have lighten/darken flavors. We would need to invert the lighten/darken for the black theme, then we could use that by using `base darken-1` for example, and it would adjust automatically whatever the theme we use.